src/index.hot.js

```diff
@@ -1,19 +1,20 @@
+/* eslint global-require: 0 */
+import 'babel-polyfill';
```

Comment:
Which one do you think is better between putting this import here vs putting this import in `` webpack.config.entry ``?